src/WebJobs.Script.Analyzers/WebJobs.Script.Analyzers.csproj

```diff
@@ -41,4 +45,7 @@
     </ItemGroup>
   </Target>
 
+  <Target Name="_AddNuGetDlls">
```

Comment:
Did you mean to have this empty target?

Reply:
Ah, thanks. This is a remnant; found a cleaner way to include the WebJobs DLLs in the analyzer nuget. Removing.